File: python/cash/cash.py
from cs50 import get_float

dollars = -1
while dollars <= 0:
    dollars = get_float('Change owed: ')

cents = dollars*100
count = 0

while cents >= 25:
    cents = cents - 25
    count += 1

while cents >= 10:
    cents = cents - 10
    count += 1

while cents >= 5:
    cents = cents - 5
    count += 1

while cents >= 1:
    cents = cents - 1
    count += 1

print(f'{count}')

File: c/cash.c
// computes the minimum number of coins a person is owed in change

#include <stdio.h>
#include <cs50.h>
#include <math.h>

float dollars;
int cents, count;

int main(void) {
    // INPUT CHANGE DUE
    do {
        dollars = get_float("Change owed: ");
    }
    while (dollars <= 0);

    // CONVERT CENTS TO INTEGER FOR PRECISION
    cents = round(dollars * 100);

    for (;cents >= 25; count++) {
        cents = cents - 25;
    }

    for (;cents >= 10; count++) {
        cents = cents - 10;
    }

    for (;cents >= 5; count++) {
        cents = cents - 5;
    }

    for (;cents >= 1; count++) {
        cents = cents - 1;
    }

    printf("%d\n", count);
}